charts/kcp/templates/certmanagerissuer.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
What do you think about creating the LE ClusterIssuers optionally, based on a value? I would assume that users already register a ClusterIssuer for LE since they might have other public workloads running. It would be nice if you could plug an existing ClusterIssuer into the chart instead.

Reply:
Okay, reviewing this I think you can already plug your own ClusterIssuer. But I would still argue that creating the LE ClusterIssuers should be optional, since it will register an account with LE that might not be needed.

Reply:
Added LE Issuers under the flags.